### PR TITLE
[WIP] Dependency Injection Asp.Net Core sample

### PR DESCRIPTION
**Pull request by mauroservienti**
Solves this problem https://gitter.im/Particular/NServiceBus?at=59d1e36fb59d55b82360cb5e
I'm not sure if we need to add a Controller showing that the same `MyService` can be injected there as well.

### CODE REVIEW

**Comment by mauroservienti** [comment]
build is failing:

```
System.Exception : Could not find nuget for alias 'aspnetcore'.
```

Give that the sample uses `Autofac` should I move it in the `Autofac` directory?

File: samples/dependency-injection/aspnetcore/core_7/Sample/Program.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        await host.StopAsync()
+            .ConfigureAwait(false);
+    }
+
+    static void AttemptToLaunchBrowser(string address)
+    {
+        Console.WriteLine($"Attempting to open browser to: {address}");
+        try
+        {
+            using (Process.Start("explorer.exe", address))
```
Comment (mauroservienti) [inline samples/dependency-injection/aspnetcore/core_7/Sample/Program.cs]: Being a net core only sample does this line makes this Windows only?

Comment (SimonCropp) [reply samples/dependency-injection/aspnetcore/core_7/Sample/Program.cs]: Yes hence the try catch

**Comment by SimonCropp** [comment]
The divergence is to allow supporting poping up the webpage

**Comment by mauroservienti** [comment]
Being a web project a web page is automatically started even with the previous configuration when the project is started e.g. from VS

**Comment by SimonCropp** [comment]
i have hit many bugs with iis express+vs2017+netcore. The current approach works around them. Happy to change it back if u want. I would just not prefer to shave the yaks

**Comment by mauroservienti** [comment]
I haven't hit any, also web projects can be run as simple console apps not hosted in IIS at all. Anyway let's leave this as is.

**Comment by SimonCropp** [comment]
Afaik if u run it in console mode it does not open the browser

**Comment by mauroservienti** [comment]
VS, change startup mode from IIS to Console, F5 and browser pops up.

**Comment by SimonCropp** [comment]
Is that without my process start change?

**Comment by mauroservienti** [comment]
yes, just change this:

image

**Pull request merged**